### PR TITLE
Add april fools prank

### PR DESCRIPTION
**Pull request by pingiun**
### Summary
Ssht don't tell anyone, this will rotate the whole website on april fools

### How to test
Steps to test the changes you made:
1. Wait until april fools
2. See the whole website is slightly rotated

### CODE REVIEW

File: website/thaliawebsite/templates/base.html [hunk 1]
```diff
@@ -40,6 +40,13 @@
             <link href="{% static "css/main.scss" %}" rel="stylesheet" type="text/x-scss">
         {% endcompress %}
     {% endblock %}
+    {% if APRIL_FOOLS %}
+    <style>
+        body {
+            transform: rotate(.5deg);
```
Comment (se-bastiaan) [inline website/thaliawebsite/templates/base.html:46]: May I suggest

```
import random
from django import template

register = template.Library()

@register.simple_tag
def random_float(a, b=None):
    if b is None:
        a, b = 0, a
    return round(random.uniform(a, b), 2)
```

with 

`{% load random_float %}`

and 

`rotate({% random_float 2 -2 %}deg);`

Comment (pingiun) [reply website/thaliawebsite/templates/base.html:46]: no, because this will be removed in the release after the next one anyway

Comment (pingiun) [reply website/thaliawebsite/templates/base.html:46]: oops sorry, didn't see what you were actually suggesting

Comment (pingiun) [reply website/thaliawebsite/templates/base.html:46]: i think it's fine the way it is though, very subtle

**Review by se-bastiaan — CHANGES_REQUESTED** [review]
Timezones bah bah bah

**Review by se-bastiaan — APPROVED** [review]

**Pull request merged**